requirements/edx/github.in

```diff
@@ -49,25 +63,15 @@
 #     re-install the package each time, and can be useful when working with two
 #     repos before picking a version number. Don't use 0.0 on master, only for
 #     tight-loop work in progress.
-
-
-# Python libraries to install directly from github
-
-# Third-party:
--e git+https://github.com/openedx/django-wiki.git@1.1.1#egg=django-wiki
--e git+https://github.com/openedx/olxcleaner.git@2f0d6c7f126cbd69c9724b7b57a0b2565330a297#egg=olxcleaner
+#
+#   * Alphabetize dependencies by DIST-NAME.
+git+https://github.com/openedx/blockstore.git@1.2.5#egg=blockstore==1.2.5
+git+https://github.com/openedx/codejail.git@3.3.0#egg=edx-codejail==3.3.0
+git+https://github.com/openedx/codejail.git@3.1.3#egg=codejail==3.1.3
+git+https://github.com/openedx/django-require.git@f4f01e4e959adc6210873ae99e7f2c3741afbf35#egg=django-require==1.0.12
+git+https://github.com/openedx/django-wiki.git@1.1.1#egg=django-wiki
```

Comment:
I wonder if pip is smart enough that we could speed up installs on these by specifying the full commit hash.

Reply:
Perhaps, but I think I'd rather spend those mental cycles just turning these into proper PyPI packages, which gives a huge speedup.